Examples/CORAL2_Benchmark/Coral_2.inp

```diff
@@ -13,6 +13,7 @@ Simulation:
    eMin: 1e-09
    nGroups: 230
    lowWeightCutoff: 0.001
+   fluence: 1
```

Comment:
Ryan,  Let's set a control on the overall correctness test, not just the fluence.